pages/samvera/manager_guide/hyrax_2.1/edit-works-2.1.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Editing a Work"
+permalink: edit-works-2.1.html
+folder: samvera/manager_guide/edit-works-2.1.md
+sidebar: home_sidebar
+a-z: ['Works - Editing a Work', 'Editing a Work - Repository Administration']
+keywords: Best Practices, managers, repo mangers, hyrax administration
+tags: [user_resources]
+categories: How to use the Administration panel in hyrax
+version:
+  id: 'hyrax_2.1-stable'
+  versions:  
+    - label: 'Hyrax 1.0'
+      link:  'edit-works-1.0.html'
+    - label: 'Hyrax 2.0'
+      link:  'edit-works-2.0.html'
+    - label: 'Hyrax 2.1'
+      link:  'edit-works-2.1.html'
+      selected: 'true'
+---
+# Editing a Work
+
+There are two views for works: Browse view and Edit view. The public-facing view of a work is the Browse view. A user will navigate to a work and then click “Edit” to access the edit view. As an admin user, you would be able to edit any work in the repository.
+
+In the edit view for a work, you can change the description, add files, edit relationships, edit sharing, and edit visibility. If you edit any of these settings, you must click “Save” in order to apply your changes. Note that if you add files, you must recheck the deposit agreement before saving.
+
+If you edit the sharing or visibility of the work, you can either opt to have all files in the work inherit these permissions, or you can opt to have the individual files retain their permissions so that they can be edited manually.
+
+Set child/parent work: In the “Relationships” tab, you can now add a child or parent work to the work - this is only available after the work is deposited, not at the time of deposit. A child work added to this work will appear as an Item in this work. A parent work will add this work to its Items. To add a child or parent work, enter the unique ID (found at ???) of the work under “Child Work” or “Parent Work” and click “Add”. Click “Save” to apply your changes.
```

Comment:
ohhh can you only do this on edit? i don't think you do the unique ID, it's a title search. or you "start on the parent attach a child"

Reply:
changed it to title